po/pt_BR.po

```diff
@@ -8714,9 +8714,8 @@ msgid "GnuCash could not find the linked document."
 msgstr "O GnuCash não conseguiu obter trava (\"lock\") para %s."
 
 #: gnucash/gnome-utils/gnc-gnome-utils.c:580
-#, fuzzy
 msgid "GnuCash could not open the linked document:"
-msgstr "O GnuCash não conseguiu obter trava (\"lock\") para %s."
+msgstr ""
```

Comment:
@farribeiro There is no need to clear msgstr. As long as fuzzy is set, msgstr is not used.

Reply:
In the weblate I deleted the translation what was wrong and sent a suggestion

Reply:
I usually use the suggestion instead of applying it as it needs editing. This to be reviewed

Reply:
A translation tip, some entries don't have the same translation, one case is this entry

Reply:
> In the weblate I deleted the translation what was wrong and sent a suggestion, and not edit po file directly

Again something learned about how weblate works. Thanks!